packages/dapp/src/components/Widget/Widget.tsx

```diff
@@ -1,7 +1,166 @@
-import { LiFiWidget } from '@lifi/widget';
-import { useWidgetConfig } from '../../hooks/';
+import { Token } from '@lifi/sdk';
+import {
+  addChain,
+  switchChain,
+  switchChainAndAddToken,
+} from '@lifi/wallet-management';
+import { HiddenUI, LiFiWidget, WidgetConfig } from '@lifi/widget';
+import { useTheme } from '@mui/material/styles';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useUserTracking } from '../../hooks';
+import { useWallet } from '../../providers/WalletProvider';
+import { LanguageKey } from '../../types';
 
 export function Widget({ starterVariant }) {
-  const widgetConfig = useWidgetConfig((starterVariant = { starterVariant }));
```

Comment:
<img src="https://media.tenor.com/HN372NlC4IgAAAAd/what-was-that-the-black-mastadonte.gif" alt="" width="200" />

Reply:
Let´s forget about this, the zustand one will take over that chunk as well